charts/hlf-k8s/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 10.2.3
+
+### Changed
+- Update image versions
```

Comment:
There are no changes in the values, does the chart pull the image version automatically from `Chart.yaml`?

Reply:
Looking more closely, my changelog is wrong. 
I just copied the one from the releasing guide. The changes I made were only about skaffold configuration files - which are not involving any Helm chart.

Reply:
Actually, the versions of the built images (made during tag or push to main) used are defined in the `values.yaml` file, more precisely [L18](https://github.com/Substra/hlf-k8s/blob/main/charts/hlf-k8s/values.yaml#L18) and [L208](https://github.com/Substra/hlf-k8s/blob/main/charts/hlf-k8s/values.yaml#L208).